Test(eos_cli_config_gen): Added test in molecule for `monitor_connectivity` to improve the coverage (#4362)
aristanetworks/ansible-avd: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

+      encryption: disabled
```

> ### `monitor_connectivity.address_only`, `monitor_connectivity.hosts[].address_only`, `monitor_connectivity.vrfs[].address_only` and `monitor_connectivity.vrfs[].hosts[].address_only` default options has been replaced with defined

Reviewer: ```suggestion
### `address_only` options under `monitor_connectivity` no longer have a default value of `true`
```

Author: updated

In AVD 4.0.0  the `monitor_connectivity.address_only`, `monitor_connectivity.hosts[].address_only`, `monitor_connectivity.vrfs[].address_only` and `monitor_connectivity.vrfs[].hosts[].address_only` options are default as true so if not defined then it always renders the address-only configuration to set the default interface for probing.

> As of AVD 5.0.0 we have to define the `monitor_connectivity.address_only`, `monitor_connectivity.hosts[].address_only`, `monitor_connectivity.vrfs[].address_only` and `monitor_connectivity.vrfs[].hosts[].address_only` options to renders the address-only configuration to set the default interface for probing.

Reviewer: The suggestion tool is not able to provide the closing tripple backticks, so add that manually.

```suggestion
The default value `true` of data models `monitor_connectivity.hosts[].address_only`, `monitor_connectivity.vrfs[].address_only` and `monitor_connectivity.vrfs[].hosts[].address_only` has been removed, so it is now required to set `address_only: true` to render `address-only` in the EOS configuration.

```diff
 monitor_connectivity:
   hosts:
     - name: server1
       description: server1_connectivity_monitor
       ip: 10.10.10.1
       local_interfaces: HOST_SET
+      address_only: true
       url: https://server1.local.com
   vrfs:
     - name: red
+      address_only: true
       ...
     - name: blue
       ...
       hosts:
         - name: server4
           description: server4_connectivity_monitor
           ip: 10.10.20.1
+          address_only: true
           local_interfaces: VRF_HOST_SET
           url: https://server2.local.com
```

```

Author: updated with addition of `monitor_connectivity.address_only`
